src/main/java/org/betonquest/betonquest/compatibility/holographicdisplays/HologramWrapper.java

```diff
@@ -0,0 +1,122 @@
+package org.betonquest.betonquest.compatibility.holographicdisplays;
+
+import lombok.CustomLog;
+import lombok.Getter;
+import me.filoghost.holographicdisplays.api.hologram.Hologram;
+import me.filoghost.holographicdisplays.api.hologram.VisibilitySettings;
+import org.betonquest.betonquest.BetonQuest;
+import org.betonquest.betonquest.api.config.QuestPackage;
+import org.betonquest.betonquest.compatibility.holographicdisplays.lines.AbstractLine;
+import org.betonquest.betonquest.id.ConditionID;
+import org.betonquest.betonquest.utils.PlayerConverter;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+
+import java.util.List;
+
+/**
+ * Wrapper class for {@link Hologram} that stores data parsed from hologram configuration inside <code>custom.yml</code>.
+ */
+@SuppressWarnings("PMD.CommentRequired")
+@Getter
+@CustomLog
+public class HologramWrapper {
+
+    /**
+     * Actual hologram
+     */
+    private final Hologram hologram;
+
+    /**
+     * Interval in ticks that lie between updates to the visibility and content
+     */
+    private final int interval;
+
+    /**
+     * Indicates whether the displayed content of the hologram is changing after a while. HolographicDisplays variables
+     * are not updated BetonQuest, it does not make a hologram flexible. However, vontent updates such as refreshing the
```

Comment:
```suggestion
     * are not updated BetonQuest, it does not make a hologram flexible. However, vrontent updates such as refreshing the
```

Reply:
I changed it to "content" since this is what should have been there